repose-aggregator/components/filters/header-translation-filter/src/main/scala/org/openrepose/filters/headertranslation/HeaderTranslationFilter.scala

```diff
@@ -36,17 +36,17 @@ import scala.collection.JavaConversions._
 class HeaderTranslationFilter @Inject()(configurationService: ConfigurationService)
   extends Filter with UpdateListener[HeaderTranslationType] with LazyLogging {
 
-  private final val DEFAULT_CONFIG = "header-translation.cfg.xml"
+  import HeaderTranslationFilter._
```

Comment:
I've been wondering, too.  I'll do some research.  My guess is that the only difference is going to be where in memory the value is stored.